lms/static/scripts/frontend_apps/utils/content-item.js

```diff
@@ -1,5 +1,3 @@
-import { stringify } from 'querystring';
```

Comment:
This should have used `query-string` (standalone package) instead of `querystring` (Browserify's polyfill of Node's `querystring`). Fortunately both packages have the same API.

Reply:
In that this branch would blow up if anything else was trying to import something from `querystring`, it seems safe (?) to assume that nothing else does.

Reply:
> In that this branch would blow up if anything else was trying to import something from querystring

Indeed. The change to the `create-bundle.js` script mean that modules are no longer able to accidentally import `querystring`.